webapp-django/crashstats/crashstats/static/crashstats/css/base/typography.less

```diff
@@ -43,7 +43,7 @@ em {
     font-style: italic;
 }
 
-b {
+strong, b {
```

Comment:
reST uses `strong`, but that wasn't showing up as bold. I didn't spend a lot of time looking into why. This fix seemed good enough for now.